node/bft/ledger-service/src/lib.rs

```diff
@@ -60,3 +67,38 @@ macro_rules! spawn_blocking {
         }
     };
 }
+
+#[cfg(feature = "metrics")]
+fn update_block_metrics<N: Network>(block: &Block<N>) {
+    use snarkvm::ledger::ConfirmedTransaction;
+
+    let accepted_deploy = AtomicUsize::new(0);
+    let accepted_execute = AtomicUsize::new(0);
+    let rejected_deploy = AtomicUsize::new(0);
+    let rejected_execute = AtomicUsize::new(0);
+
+    // Add transaction to atomic counter based on enum type match.
+    block.transactions().par_iter().for_each(|tx| match tx {
```

Comment:
`cfg_iter!`?

Reply:
@miazn for context `cfg_iter!` is a Rust macro within SnarkVM that will iterate through a collection either in parallel or serial depending on the feature flag. 

A usage example is can be found here [here](https://github.com/AleoHQ/snarkVM/blob/d48f6fb7d7ca36d134407baebcc59952fe58c546/ledger/block/src/verify.rs#L245-L255). 

The actual source code behind this macro is [here](https://github.com/AleoHQ/snarkVM/blob/d48f6fb7d7ca36d134407baebcc59952fe58c546/utilities/src/parallel.rs#L88-L98).

Since there exists a `serial` flag that will disable parallel iteration. It's important to use this macro to ensure that this code will still run if the `serial` flag is enabled.

Reply:
ah gotcha, wasn't privy to that -- changed :D